content/actions/learn-github-actions/migrating-from-azure-pipelines-to-github-actions.md

```diff
@@ -37,7 +37,7 @@ When migrating from Azure Pipelines, consider the following differences:
 
 - Azure Pipelines supports a legacy _classic editor_, which lets you define your CI configuration in a GUI editor instead of creating the pipeline definition in a YAML file. {% data variables.product.prodname_actions %} uses YAML files to define workflows and does not support a graphical editor.
 - Azure Pipelines allows you to omit some structure in job definitions. For example, if you only have a single job, you don't need to define the job and only need to define its steps. {% data variables.product.prodname_actions %} requires explicit configuration, and YAML structure cannot be omitted.
-- Azure Pipelines supports _stages_ defined in the YAML file, which can be used to create deployment workflows. {% data variables.product.prodname_actions %} requires you to separate stages into separate YAML workflow files.
```

Comment:
Usługa Azure Pipelines obsługuje _stages_ zdefiniowane w pliku YAML, który może służyć do tworzenia przepływów pracy wdrażania. {% danych variables.product.prodname_actions %} wymaga oddzielenia etapów na oddzielne pliki przepływu pracy YAML.

Reply:
Hi - could you please translate your feedback in english? Thanks

Reply:
Eveything looks great well done ✅

Reply:
Xin chào